Furball.Vixie.TestApplication/Tests/TestQuadRendering.cs

```diff
@@ -14,10 +14,10 @@ public class TestQuadRendering : Screen {
     private Texture       _texture;
 
     public override void Initialize() {
-        this._vixieRenderer = new Renderer();
+        this._vixieRenderer = new Renderer(TestGame.Instance.WindowManager.GraphicsBackend);
```

Comment:
Is there anything stopping you from making you *not* have to pass in TestGame.Instance.WindowManager.GraphicsBackend everywhere? like before this we had it automatically get the current one, while yes we're trying to remove state I'd much rather have a cleaner API than having to pass stuff around everywhere, even if it isn't user facing code